drivers/SmartThings/zwave-switch/profiles/rgbw-bulb-2700K-6500K.yml

```diff
@@ -0,0 +1,22 @@
+name: rgbw-bulb-2700K-6500K
+components:
+- id: main
+  capabilities:
+  - id: switch
+    version: 1
+  - id: switchLevel
+    version: 1
+  - id: colorTemperature
+    version: 1
+    config:
+      values:
+        - key: "colorTemperature.value"
+          range: [ 2700, 6500 ]
```

Comment:
You can now simply define the range your bulb supports in code using the `colorTemperatureRange` attribute.

Example: https://github.com/SmartThingsCommunity/SmartThingsEdgeDrivers/blob/main/drivers/SmartThings/matter-switch/src/generic_handlers/attribute_handlers.lua#L221

Reply:
@greens 
Should I do something like this during device initialization then?

device:emit_event(capabilities.colorTemperature.colorTemperatureRange({ value = {minimum = 2700, maximum = 6500} }))
device:emit_event(capabilities.colorTemperature.colorTemperature(6500))

Then, should I change the profile to rgbw-bulb.yml and take out the config section?